[WIP] Rolfes third branch

rolfedh opened the pull request:
Updated commands to send commands to the Swarm manager instead of to Docker Engine host where the Swarm manager resides.

Signed-off by Rolfe Dlugy-Hegwer <rollfedh-contractor@docker.com>

rolfedh commented:
THIS PR IS FOR WIP REVIEW

nishanttotla commented:
Since this doc is under review, can we change to using standard ports for Swarm (3375 for non-TLS instead of 2360 or others) so that we don't confuse people?

cc @dongluochen @moxiegirl

vieux commented on docs/install-w-machine.md:
```diff
@@ -101,64 +101,113 @@ Here, you connect to each of the hosts and create a Swarm manager or node.
-        agent1    -        virtualbox   Running   tcp://192.168.99.102:2376           v1.9.1   
-        agent2    -        virtualbox   Running   tcp://192.168.99.103:2376           v1.9.1   
-        manager   *        virtualbox   Running   tcp://192.168.99.100:2376           v1.9.1   
+        node1_host    -        virtualbox   Running   tcp://192.168.99.102:2376           v1.10.0   
```
spacing is wrong here

rolfedh commented:
Thanks for reviewing, Nishant! I'll do that.

On Fri, Feb 5, 2016 at 2:59 PM, Nishant Totla <notifications@github.com>
wrote:

> Since this doc is under review, can we change to using standard ports for
> Swarm (3375 for non-TLS instead of 2360 or others) so that we don't confuse
> people?
>
> cc @dongluochen <https://github.com/dongluochen> @moxiegirl
> <https://github.com/moxiegirl>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/swarm/pull/1763#issuecomment-180534593>.
>

rolfedh commented:
I fixed the spacing issues and changed the mapped port number to 3375.

I could really use your help fixing the TLS authentication issues that are
causing the Swarm nodes to show up in the Swarm master's info with "Error:
Cannot connect to the docker engine endpoint".


On Sat, Feb 6, 2016 at 8:35 AM, Rolfe Dlugy-Hegwer <rolfedh@gmail.com>
wrote:

> Thanks for reviewing, Nishant! I'll do that.
>
> On Fri, Feb 5, 2016 at 2:59 PM, Nishant Totla <notifications@github.com>
> wrote:
>
>> Since this doc is under review, can we change to using standard ports for
>> Swarm (3375 for non-TLS instead of 2360 or others) so that we don't confuse
>> people?
>>
>> cc @dongluochen <https://github.com/dongluochen> @moxiegirl
>> <https://github.com/moxiegirl>
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/docker/swarm/pull/1763#issuecomment-180534593>.
>>
>
>

nishanttotla commented on docs/install-w-machine.md:
```diff
@@ -101,64 +94,114 @@ Here, you connect to each of the hosts and create a Swarm manager or node.
+          └ Reserved CPUs: 0 / 0
+          └ Reserved Memory: 0 B / 0 B
+          └ Labels:
+          └ Error: Cannot connect to the docker engine endpoint
```
Both your nodes here are unreachable. Is this intentional?

nishanttotla commented:
@rolfedh we fixed some immediate issues with some existing docs, so your branch now has merge conflicts which you need to resolve. I would recommend dropping changes that conflict with #1778.

cc @dongluochen @moxiegirl

rolfedh commented:
@nishant @dongluochen @moxiegirl Thank you for responding to/resolving the
issue I raised about the nodes. I'll manually merge my changes into the
updated file.

Rolfe

On Tue, Feb 9, 2016 at 6:04 PM, Nishant Totla <notifications@github.com>
wrote:

> @rolfedh <https://github.com/rolfedh> we fixed some immediate issues with
> some existing docs, so your branch now has merge conflicts which you need
> to resolve. I would recommend dropping changes that conflict with #1778
> <https://github.com/docker/swarm/pull/1778>.
>
> cc @dongluochen <https://github.com/dongluochen> @moxiegirl
> <https://github.com/moxiegirl>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/swarm/pull/1763#issuecomment-182126429>.
>

moxiegirl commented:
test please

rolfedh commented:
test this please

rolfedh commented:
test this please

rolfedh commented:
test this please

rolfedh commented:
test this please

rolfedh commented:
test this please
